independent-projects/resteasy-reactive/client/runtime/src/main/java/org/jboss/resteasy/reactive/client/handlers/ClientSendRequestHandler.java

```diff
@@ -166,6 +166,10 @@ public void handle(Throwable failure) {
         requestOptions.setURI(uri.getPath() + (uri.getQuery() == null ? "" : "?" + uri.getQuery()));
         requestOptions.setFollowRedirects(followRedirects);
         requestOptions.setSsl(isHttps);
+        Object readTimeout = state.getConfiguration().getProperty(QuarkusRestClientProperties.READ_TIMEOUT);
+        if ((readTimeout instanceof Long)) {
```

Comment:
`instanceof` also means that `readTimeout` is not `null` (and IntelliJ hints as much if one tries to add the null check)

Reply:
if you do a rework, you have double brackets in the `if`

Reply:
Good point, let me fix